### PR TITLE
Refactoring localAddressMapping in AbstractRestFitness

### PR DESCRIPTION
**Pull request by seran**

### CODE REVIEW

File: e2e-tests/spring-rest-openapi-v2/src/test/java/org/evomaster/e2etests/spring/examples/wiremock/service/ExternalServiceMockingFlakyEMTest.java [hunk 1]
```diff
@@ -48,6 +50,8 @@ public void externalServiceMockingTest() throws Throwable {
                     args.add("USER");
                     args.add("--externalServiceIP");
                     args.add("127.0.0.5");
+                    args.add("--minimize");
```
Comment (arcuri82) [inline e2e-tests/spring-rest-openapi-v2/src/test/java/org/evomaster/e2etests/spring/examples/wiremock/service/ExternalServiceMockingFlakyEMTest.java:53]: if test disabled, shouldn't this be removed? u forgot to remove it, or was it added on purpose?

File: e2e-tests/spring-rest-openapi-v3/src/test/kotlin/org/evomaster/e2etests/spring/openapi/v3/wiremock/okhttp3/WmHttpOkHttp3EMTest.kt [hunk 2]
```diff
@@ -51,13 +51,13 @@ class WmHttpOkHttp3EMTest : SpringTestBase() {
...
                 assertHasAtLeastOne(solution, HttpVerb.GET, 200, "/api/wm/socketconnect/string", "Hello There")
-                assertHasAtLeastOne(solution, HttpVerb.GET, 400, "/api/wm/socketconnect/string", null)
-                assertHasAtLeastOne(solution, HttpVerb.GET, 500, "/api/wm/socketconnect/string", null)
+                assertHasAtLeastOne(solution, HttpVerb.GET, 400, "/api/wm/socketconnect/string", "")
+                assertHasAtLeastOne(solution, HttpVerb.GET, 500, "/api/wm/socketconnect/string", "")
 
                 assertHasAtLeastOne(solution, HttpVerb.GET, 200, "/api/wm/socketconnect/object", "OK")
             },
-            3
+            10
```
Comment (arcuri82) [inline e2e-tests/spring-rest-openapi-v3/src/test/kotlin/org/evomaster/e2etests/spring/openapi/v3/wiremock/okhttp3/WmHttpOkHttp3EMTest.kt:59]: hmmm... unsure about this. ll need to discuss

**Review by arcuri82 — CHANGES_REQUESTED** [review]

File: e2e-tests/spring-rest-openapi-v3/src/test/kotlin/org/evomaster/e2etests/spring/openapi/v3/wiremock/okhttp3/WmHttpOkHttp3EMTest.kt (hunk 2 above)
Comment (seran) [reply e2e-tests/spring-rest-openapi-v3/src/test/kotlin/org/evomaster/e2etests/spring/openapi/v3/wiremock/okhttp3/WmHttpOkHttp3EMTest.kt:59]: Forget to clean it.

File: e2e-tests/spring-rest-openapi-v2/src/test/java/org/evomaster/e2etests/spring/examples/wiremock/service/ExternalServiceMockingFlakyEMTest.java (hunk 1 above)
Comment (seran) [reply e2e-tests/spring-rest-openapi-v2/src/test/java/org/evomaster/e2etests/spring/examples/wiremock/service/ExternalServiceMockingFlakyEMTest.java:53]: fixed it

**Review by arcuri82 — APPROVED** [review]

**Pull request merged**